packages/nimble-components/src/table-column/text/template.ts

```diff
@@ -7,9 +7,17 @@ TableCellState<TableColumnTextCellRecord, TableColumnTextColumnConfig>
 >`
     <span
         class="${x => (typeof x.cellRecord.value === 'string' ? '' : 'placeholder')}"
+        @mouseover="${(_x, c) => {
+        const span = c.event.target as HTMLSpanElement;
+        if (span?.textContent && span.offsetWidth < span.scrollWidth) {
+            span.setAttribute('title', span.textContent);
```

Comment:
In storybook it looks like the title for truncated cells has extra padding on the top, left, and bottom. I'm seeing this in Safari, Edge, and Firefox (didn't try Chrome). I don't see the same for the title that we set on the column header.

I'm not sure what would cause this but I did notice in dev tools that the title has some whitespace around it (`title="  I'm number 10!  "`). Perhaps instead of reading the span's contents we should use the same string value/placeholder that was used to originally populate it.

<img width="522" alt="image" src="https://user-images.githubusercontent.com/10500124/222784440-b0483cd4-1d39-496d-869f-d4533792aa36.png">

<img width="522" alt="image" src="https://user-images.githubusercontent.com/10500124/222784520-016f9763-d1ea-42ca-aaa2-9a27a2fa3fd5.png">

Reply:
Looks like Chrome trims whitespace automatically, whereas the other browsers don't. Now explicitly trimming.